autofaiss/indices/distributed.py

```diff
@@ -474,15 +473,13 @@ def _create_and_train_index_from_embedding_dir() -> TrainedIndex:
             id_columns=id_columns,
         )
 
-        index_output_root_dir = os.path.join(temp_root_dir, "training", partition)
+        index_output_root_dir = os.path.join(temp_root_dir, "training", str(uuid4()))
         output_index_path = save_index(trained_index.index_or_path, index_output_root_dir, "trained_index")
-        return TrainedIndex(output_index_path, trained_index.index_key, embedding_root_dir)
-
-    partition = extract_partition_name_from_path(embedding_root_dir)
+        return TrainedIndex(output_index_path, trained_index.index_key, embedding_root_dirs)
 
     if not index_path:
         # Train index
-        rdd = ss.sparkContext.parallelize([embedding_root_dir], 1)
+        rdd = ss.sparkContext.parallelize([13], 1)
```

Comment:
13 ?

Reply:
The value does not matter. I create a fake partition just to train the index on an executor and not on the driver. After discussion with Victor, we thought it was weird to use embedding_root_dir so I'm using a magic number instead. But maybe it is confusing as well. I will put a comment.